docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,201 @@
+-  If an issue is unclear, then try to edit the title and description for more clarity or leave a comment requesting that edits to the issue be made.
+
+> **Note:** Above rules are applicable to both new and existing issues.
+
+### Dealing with missing information
+
+Depending on the issue, you might not feel all this information is needed. Use your best judgement. If you cannot triage an issue using what its author provided, explain kindly to the author that they must provide the above information to clarify the problem. Label issue with `triage/needs-information`.
+
+If the author provides the standard information but you are still unable to triage the issue, then request additional information. Do this kindly and politely because you are asking for more of the author's time. Label issue with `triage/needs-information`. `triage/needs-information`.
+
+If the author does not respond to the requested information within the timespan of a week, close the issue with a kind note stating that the author can request for the issue to be reopened when the necessary information is provided.
+
+If you receive a notification that additional information was provided but you are no longer working on issue triage, then you should delegate the issue to the current person working on issue triage.
+
+## 3. Categorizing an issue
+
+### Duplicate issues
+
+Make sure it's not a duplicate by searching existing issues using related terms from the issue title and description. If you think the issue may be a potential duplicate and can't find the original existing issue, then please reach out to one of the maintainers and ask for help. If you identify that the issue is a duplicate of an existing issue:
+
+1. Add a comment `duplicate of #<issue number>`
+2. Add the `triage/duplicate` label
+
+### Bug reports
+
+If it's not perfectly clear that it's an actual bug, quickly try to reproduce it.
+
+**It's a bug/it can be reproduced:**
+
+1. Add a comment describing detailed steps for how to reproduce it, if applicable.
+2. If maintainers aren't able to address the issue in a timely manner, then label the issue with `help wanted` and an optional `beginner friendly`. Also, include pointers to the problematic source code. Doing this allows us to solicit help from the community to get the issue resolved.
+3. Move on to [prioritizing the issue](#4-prioritization-of-issues).
+
+**It can't be reproduced:**
+
+1. Either request that [more information](#2-ensure-the-issue-contains-basic-information) is needed to investigate the issue more thoroughly. Provide details in a comment.
+or
+2. Either [delegate further investigations](#investigation-of-issues) to someone else.  Provide details in a comment.
+
+**It works as intended/by design:**
+
+1. Kindly and politely add a comment explaining briefly why we think it works as intended and close the issue.
+2. Label the issue `triage/works-as-intended`.
+3. Remove the `needs-triage` label.
+
+**It does not work as intended/by design:**
```

Comment:
Please add some description here or at least add a comment that indicates an action item for maintainers to provide the additional information. There is already a suitable matching action described for feature requests (prioritization and removal of the 'needs-triage' label). That could be copied here to complete this section.

Reply:
We will discuss with server storage team and get back to you.

Reply:
If something is both "not a bug" but also "does not work as intended/by design" then it seems appropriate to convert/tag the issue into a Feature Request. Perhaps you can add language suggesting this.

Reply:
@jeff-yin we will discuss with server storage team and will get back to you.

Reply:
Yes, this comment was discussed with server/storage team. It is agreed that "It does not work as intended/by design" is same as "It's a bug/it can be reproduced". Hence the additional category/line "It does not work as intended/by design" has been removed.